### DISCUSSION TITLE
Circular Track map and projected position after pitting

### DISCUSSION THREAD

**Discussion by TobyDS** [f1multiviewer/issue-tracker#467] [Feature requests] [answered]
It would be great if we could have a circular track map to help visualise driver pit position/time loss. By inputting the average pit time loss an estimated position after pitting could be calculated for each driver and given as a column that could be added to any of the live timing views, or perhaps to the overlay on the cockpit cams.

The circular track map could then be offered as another track overlay and perhaps a driver could be selected on it which would then show where they would be relative to the other drivers were they to stop. Lapped cars could then be shown on additional radii of the track map to indicate that they are lapped. For reference I have added an image showing the way in which this was done on the F1 delayed live timing chrome extension.

![2g2fefdz1ny71](https://github.com/f1multiviewer/issue-tracker/assets/33652695/dcf8e5ab-995f-4f31-83aa-4a54b125fec0)

Finally, not only could you allow a pit stop time loss for green flag racing to be added, but you could also allow a time loss under safety car / VSC to also be added and whenever either form of safety car is deployed, automatically switch to use that time loss for the position calculations / visualisations.

This addition could massively add to the users awareness and understanding of the pit strategies during the race and I think that the projected position after pit wouldn't be too difficult to implement. The circular track map would be harder, but my god would it look cool and be useful.

**Comment by LapsTimeOFF** [comment, marked as answer]
Hey, this feature is now available to sponsors (https://muvi.gg/donate), and will be made globally available some time in the upcoming races
